IN_SOPT_Second_Task/IN_SOPT_Second_Task/KakaoProfileViewController.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  KakaoProfileViewController.swift
+//  IN_SOPT_Second_Task
+//
+//  Created by 박의서 on 2022/10/12.
+//
+
+import UIKit
+
+class KakaoProfileViewController: UIViewController {
+    
+    // MARK: - Profile
+    private let profileView: UIView = {
+       let view = UIView()
+        return view
+    }()
```

Comment:
```suggestion
    private let profileView = UIView()
```
초기화 시점에서 따로 지정할 속성이 없다면 위와 같이 수정해도 될 것 같습니당